docs/faq.rst

```diff
@@ -516,15 +516,30 @@ When I do a ``borg extract``, after a while all activity stops, no cpu usage,
 no downloads.
 
 This may happen when SSH terminates the connection on server side. You can
```

Comment:
Hmm, not sure that it is really the case when "SSH terminates the connection on the server side".

If the connection is really terminated, I guess the client would notice that?

So, shall we maybe say a bit more generically "This may happen when the ssh connection is stuck somehow."?

Reply:
OK